app/views/employees/_form.html.erb

```diff
@@ -49,11 +49,23 @@
     <%= f.select :service_id, options_for_select(session[:services].map{ |s| [s["name"], s["id"]]}), class: "field__input" %>
 </div>
 
-<div class="field">
-    <%= f.label :role, "Job title or role", class: "field__label" %>
-    <%= f.text_field :role, class: "field__input" %>
+<div class="field field--required">
+    <%= f.label :job_title, "Job title", class: "field__label" %>
+    <%= f.select :job_title, job_title_name.map { |title| title}, required: true, class: "field__input" %>
```

Comment:
I'm not sure you need the map on this line, since we aren't modifying the array. We can just pass in the job_title array as it is.